Used ThemeProvider for fontSize

Saksham294 opened the pull request:
With reference to the PR #58 
I have used MUI sx props for styling the "Sounds of the Salish Sea" section of the Learn page.

paulcretu commented on src/pages/learn.jsx:
```diff
@@ -32,19 +35,32 @@ export const learn = () => {
         <li>Souther Resident Killer Whale Call Catalog</li>
         <li>Exhibits</li>
       </ul>
+      <ThemeProvider theme={theme}>
```
```suggestion
```

No need to add new ThemeProviders, there's already one higher up in the render tree ([see _app.tsx](https://github.com/orcasound/orcahome/blob/main/src/pages/_app.tsx#L28))

Saksham294 replied:
Done @paulcretu

paulcretu commented on src/styles/Learn.module.css:
```diff
@@ -20,7 +20,7 @@
   text-align: left;
 }
 .imageWrapper {
-  text-align: center;
+  margin-left: 84px;
```
Since we're trying to move away from CSS modules, it would be better to put this directly in the learn page using `Box` and `sx prop`

Saksham294 replied:
Done @paulcretu

paulcretu reviewed (changes requested):
@Saksham294 thanks for the changes! Left you some comments. Also it looks like CI is failing, take a look at the checks. You should probably run `npm run lint` and `npm run format`, also make sure you've done `npm install` because it seems like you might not have all husky and lint-staged installed

Saksham294 commented:
> @Saksham294 thanks for the changes! Left you some comments. Also it looks like CI is failing, take a look at the checks. You should probably run `npm run lint` and `npm run format`, also make sure you've done `npm install` because it seems like you might not have all husky and lint-staged installed

I did "npm install' and it stated "husky-git hooks installed, changed 3 packages" . However I didn't understand what went wrong. Like I just updated files : learn.jsx,theme.ts,Learn.module.css. There was nothing related to lint, format or husky files. I also don't know anything about these files. 
So I don't know if my code will now pass the checks. Can you help me please?

paulcretu commented:
You probably started working on this feature using an old version of the project, before I set up `husky`. Husky + lint-staged automatically runs lint and format when you make a git commit, and will reject it if anything is wrong (you can [read about it here](https://github.com/okonet/lint-staged)).  When you merged in the new code, if you didn't run `npm install`, husky and lint-staged wouldn't have been installed, so your code didn't get formatted and linted.

So to fix it, you should run `npm install`, and then do `npm run format` and `npm run lint`. Those commands should make some formatting fixes to your files, which you will then need to commit and push to this PR.

Also, as an extra tip, if you're using VS Code, take a look at the `.vscode` folder. In there, there's a file named [`settings.default.json`](https://github.com/orcasound/orcahome/blob/main/.vscode/settings.default.json). If you rename/copy that into `settings.json` (put it in that same folder), VS Code will automatically run lint + format every time you save a file, so you never have to worry about it.

Saksham294 commented:
I think now it is fine. Can you check? Thanks a lot for all the support

paulcretu commented on src/pages/learn.jsx:
```diff
@@ -32,23 +34,38 @@ export const learn = () => {
+        sx={{ ml: '85px', marginTop: '30px', marginBottom: '29px' }}
+      >
+        <Typography variant="body1">
+          <p className={learnStyles.salishsea}>
```
```suggestion
```
Same comment as above, we should just use sx prop and take this opportunity to remove some CSS module classes

paulcretu commented on src/styles/theme.ts:
```diff
@@ -29,6 +29,16 @@ const theme = createTheme({
   },
   typography: {
     fontFamily: 'Montserrat',
+    h1: {
+      fontFamily: 'Mukta',
+      fontSize: '40px',
+      fontWeight: '600',
+    },
+    body1: {
+      fontFamily: 'Montserrat',
+      fontSize: '20px',
+      fontWeight: '600',
+    },
```
I think these font sizes are way too large. Just take a look at the designs
<img width="745" alt="image" src="https://user-images.githubusercontent.com/1069628/159136786-b9f188ab-8714-4de7-93c9-043173748831.png">

vs how this PR looks on my screen
<img width="1569" alt="image" src="https://user-images.githubusercontent.com/1069628/159136809-aca372d7-d803-43ee-b65f-a0c7838fe194.png">

I think we should probably just leave this as it was before because it was close enough, and we can make these kind of adjustments later on.

Saksham294 replied:
I used the Container component and put the entire section within that container. Here's an image of the same, is it fine? 
<img width="1013" alt="Screenshot 2022-03-20 at 3 28 14 PM" src="https://user-images.githubusercontent.com/86240862/159157142-3e5857f1-69d0-41b7-9d5b-f95465c383f7.png">

Saksham294 replied:
Though the design is a bit different from the one on the Figma page. Earlier I tried to match values of the styles as mentioned on the Figma design page.Like margin of the paragraph from the image was 'x' px on the Figma design so I copied the same. But it looks different with this PR as you mentioned above.

Saksham294 replied:
The section size is not like before, it is somewhat smaller in my opinion, while using the Container component.  I don't know if it is optimum . I put the header, paragraph and the image inside one single Container component. I tried using the maxWidth and sx props of the Container component but no success. Am I doing something wrong?

Saksham294 replied:
<img width="1041" alt="Screenshot 2022-03-23 at 4 10 56 PM" src="https://user-images.githubusercontent.com/86240862/159710558-97c3ecf4-cf83-4078-8a1b-f8b4cf6ae606.png">

I wrapped this section in a Container component and added individual font sizes for the text using the sx prop(removed the fontSize from the theme.ts file). I think the size is small even though I used the maxWidth="xl". I also removed marginLeft from Box components and applied it to just the Container component. Can you check if it is fine now?

paulcretu commented on src/pages/learn.jsx:
```diff
@@ -32,23 +34,38 @@ export const learn = () => {
+        </Typography>
+      </Box>
+
+      <Box sx={{ marginLeft: '84px' }}>
```
Same as the comment before, all these magical pixel values are not the best way to accomplish the effect of a left gutter

paulcretu reviewed (changes requested):
The checks look good, thanks for the changes! Looks like you removed `settings.default.json`, I re-added it (others will want it too). Also if you're not using a CSS class, might as well remove it. I went ahead and just made the changes because they're minor. However, I don't think this is ready to merge yet, left you some more comments

UXBrendan commented:
@Saksham294 

UI checks pass. Does this PR also cover the interactive graphic? If so, please link "Sounds of the Salish Sea" to this URL: https://orcasound.net/ed/booth/local.html?learn

UXBrendan commented:
@Saksham294 

We are going to deploy all passed PRs this Tuesday. Would it be possible for you to pass this PR so it's ready for our deployment session on Tuesday, Oct. 17?
